client/app/assets/less/inc/table.less

```diff
@@ -103,7 +103,7 @@
     padding-top: 5px !important;
   }
 
-  .btn-favourite,
+  .btn-favorite,
```

Comment:
Does this come from ant, or is this just us? It feels like it could potentially break stuff, not sure you want to do it in this PR.

Reply:
It's ours!